### ISSUE TITLE
Issue when same text which is to be mentioned already in the content

### ISSUE THREAD

**Issue by ShriVishnu** [podio/jquery-mentions-input#104]
Hey, When use the below example comes an issue

Hello john, Pls check this @john.

The mentioning should be done only to the second name(since @ should trigger)

But actually the regex get replaced to the first match,,, This should not be in such way ?

**Comment by ShriVishnu** [comment]
@djhvscf Hi, the issue is not about allowing repeat mentions. I am trying to add a plain text name.
For example, consider a name as just 'a'.

Now when an input text added as, "Yesterday, a comment has been added by @a"

Here, note that the first 'a' is plain text and the second 'a' is a mention of username!

Now, the first plain text is taken as the mentioned text and not the actual user name.

This is the issue which I pointed out and not an option for repeat mention!

Thanks for your kind,
Regards.

**Comment by djhvscf** [comment]
Yes, your right.. I already test it with allowRepeat option and it is happening.. I will reopen this issue..
Thanks

**Comment by sharjeelrajput** [comment]
No update on this .. Strange .. Without solving that issue that library is not useful
